File: api/Ganymede.Api.Data/Common/Language.cs
using Ganymede.Api.Data.Monsters.OptionalStats.Languages;
using System.Collections.Generic;

namespace Ganymede.Api.Data.Common
{
    public class Language
    {
        public int ID { get; set; }
        public string Name { get; set; }

        public virtual ICollection<MonsterLanguage> MonsterLanguages { get; set; }
    }
}

File: src/app/models/core/language.ts
export class Language {
	public id: number;
	public name: string;

	public static getDefault(): Language {
		const language = new Language();

		language.id = -1;

		return language;
	}

	public static isEqual(a: Language, b: Language): boolean {
		if ((!a && b) || (a && !b)) return false;
		if (!a && !b) return true;

		return a === b || (a.id === b.id && a.name === b.name);
	}
}
